components/abtesting/optimizelyXExperiment/src/optimizely-x.js

```diff
@@ -0,0 +1,65 @@
+import ExperimentUseCase from './use-case'
+
+class OptimizelyXExperimentsService {
+  /**
+   * Get OptimizelyX SDK instance
+   * @return Promise<Object>
+   */
+  static async getSDK () {
+    return window && window.optimizely
+  }
+
+  /**
+   * Get info of an experiment
+   * @param {String} experimentId
+   * @return Promise<Object>
+   * @example
+   * {
+   *  id: '100',
+   *  isActive: true,
+   *  isInExperimentHoldback: true,
+   *  variation: {
+   *      id: '101',
+   *      name: 'Variation #1'
+   *  }
+   * }
+   */
+  static async getInfo (experimentId) {
+    let sdk = await this.getSDK()
```

Comment:
I personally prefer use `const` when the value of the variable is not gonna change, as it will enable your lint to help you to identify some problems (mutating for example variables that you don't want). In the long term it might have a performance boost but right now that's not important as all of that gets compiled. :D

Not important comment. :)